hutch_python/happi.py

```diff
@@ -62,7 +63,12 @@ def get_happi_objs(db, hutch):
             logger.warning("No items found in database for %s",
                            beamline.upper())
     # Instantiate the devices needed
-    dev_namespace = load_devices(*containers, pprint=False)
+    sig = inspect.signature(load_devices)
+    if "include_load_time" in sig.parameters:
```

Comment:
We do this check in case Happi hasn't been updated?  Once it has been do we need to inspect `load_devices`?

Reply:
In my head it was "what if we have a dev install of happi without the parameter (updated hutch-python but not happi)". Despite being the one to propose it, I think this is overkill, and would be happy to remove it if you two agree.

Reply:
Normally this would be overkill but we also have arbitrary dev installs and I've been bitten in the past by this

Reply:
Eh, it's probably OK to assume updated happi, we should go tag it soon

Reply:
I'm all for being safe, I guess the question becomes more of what extent to be backward compatible.  I'm fine leaving it as well, fwiw

Reply:
Future selves: we decided that the lazy answer was the right one, but we can come back and nuke this down the line.